.github/workflows/build.yml

```diff
@@ -0,0 +1,29 @@
+name: Build
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+      - name: install dependencies
```

Comment:
That also like npm install

Reply:
Should I change "npm ci" to "npm install" ?